drivers/hmis/app/graphql/types/hmis_schema/assessment.rb

```diff
@@ -136,6 +136,16 @@ def enrollment
       load_ar_association(object, :enrollment)
     end
 
+    def load_last_user_from_versions(object)
+      # Don't need to load from versions since assessments have updated_by as an attribute on the record
+      object.updated_by
+    end
+
+    def load_created_by_user_from_versions(object)
+      # Don't need to load from versions since assessments have created_by as an attribute on the record
+      object.created_by
```

Comment:
should use a data loader to avoid n+1?

```suggestion
    def load_last_user_from_versions(object)
      # Don't need to load from versions since assessments have updated_by as an attribute on the record
      load_ar_association(object, :updated_by)
    end

    def load_created_by_user_from_versions(object)
      # Don't need to load from versions since assessments have created_by as an attribute on the record
      load_ar_association(object, :created_by)
```

Reply:
Thanks, yes!